ROADMAP.md

```diff
@@ -0,0 +1,156 @@
+
+Overall Goal: Rely heavily on data stored in the CaseRecorder for all visualization. 
+This makes the visualization more portable (i.e. can share case db with other) and 
+also makes it available after a model was run. 
+
+## 1) Metamodel viewer   
+
+### Goal:
+    - Allow users to inspect their MetaModel to check its accuracy/smoothness/etc 
+    - Encourage greater use of OM MetaModel components 
+
+### Potential Challenges:
+    - Current implementation based on Bokeh, and performance isn't as fast as we'd like yet 
+    - Some functionality requires live prediction from MetaModel instance, 
+      so we require a server process running in background. 
+      This means the functionality can't be built into the CaseRecorder
+
+### Notes: 
+    - initial capability was released in OpenMDAO V2.9.0
+
+
+## 2) Improved n2 model viz tool  
+### Goal:
+    - Make the tool more intuitive for new users 
+    - Make the tool more useful for navigating models with deep hierarchies 
+      and large number of components/variables 
+
+### Potential Challenges:
+    - Current user interface is stretched to its limit, and can't 
+      integrate any expanded navigational features. We'll need a new concept 
+      for the UI 
+
+### Notes:
+    - two different proposals for new UI concept are outlines in POE-001 and POE-002
+
+## 3) OVIS application for quickly plotting data from CaseRecorder data bases
+### Goal: 
+    - Make it simpler for users to inspect, navigate, and plot data from the case recorder 
+
+### Potential Challenges: 
+    - Going to be a separate stand-alone application. Will users be willing to download separate app? 
+    - Separate application brings large development overhead for dev-team, 
+      which may not be sustainable long term
+
+----------------------------------------------
+----------------------------------------------
+----------------------------------------------
+# New Differentiation Tools that Reduce User Effort
+
+Motivation: While analytic derivatives provide massive performance gains, 
+they also require significant user effort to implement. 
+This creates a high activation energy that prevents many users from taking advantage of the most powerful feature of OpenMDAO. 
+
+Overall Goal: Lower the effort required to implement analytic partial derivatives for components, 
+offering a spectrum of options that can potentially trade required user effort with computational efficiency. 
+
+## 1) Coloring applied to approximated partial derivatives 
+### Goal: 
+    - usable for FD and CS approximations 
+    - offers much higher performance for components with very sparse partial derivative Jacobians 
+      (e.g. vectorized components with no interaction between vector elements)
+    - When used with CS, effectively offers a fast-forward mode AD
+
+### Potential Challenges: 
+    - computational cost of coloring may be excessive for models with lots of instances 
```

Comment:
`lots of` sounds a bit more informal than `a lot of`.. might go with the latter?